app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      # Summary: Create a book suggestion
+      def create
+        book_suggestion = BookSuggestion.new(creation_params)
+        return invalid_params unless book_suggestion.save
```

Comment:
Something that I forgot to comment in the other PR where we applied this logic is that the `book_suggestion` saves the errors in the creation or update inside `book_suggestion.errors` so it's better to return those errors in the API response, that way the user knows what's wrong.
For example, you could call `invalid_params` passing the errors as a parameter and that function could render the particular error.
Try it out and see if it works better